crypto/fipsmodule/ml_kem/ml_kem_ref/indcpa.c

```diff
@@ -303,6 +314,17 @@ void indcpa_enc(ml_kem_params *params,
   poly_reduce(&v);
 
   pack_ciphertext(params, c, &b, &v);
+
+  // FIPS 203. Section 3.3 Destruction of intermidiate values.
+  OPENSSL_memset(seed, 0, sizeof(seed));
+  OPENSSL_memset(&sp, 0, sizeof(sp));
+  OPENSSL_memset(&pkpv, 0, sizeof(pkpv));
+  OPENSSL_memset(&ep, 0, sizeof(ep));
+  OPENSSL_memset(&at, 0, sizeof(at));
```

Comment:
The zeroization of at seems inconsistent with how a is zeroized in the function above. Both are arrays of polyvecs, no?